jans-config-api/docs/jans-config-api-swagger.yaml

```diff
@@ -4304,16 +4372,33 @@ components:
         umaRestrictResourceToAssociatedClient:
           type: boolean
           description: Restrict access to resource by associated client.
+        statTimerIntervalInSeconds:
+          type: integer
+          description: Statistical data capture time interval.
+        statAuthorizationScope:
+          type: string
+          description: Scope required for Statistical Authorization. 
         allowSpontaneousScopes:
           type: boolean
-          description: Specifies whether to allow spontaneous scopes
+          description: Specifies whether to allow spontaneous scopes.
         spontaneousScopeLifetime:
           type: integer
           description: The lifetime of spontaneous scope in seconds.
         openidSubAttribute:
           type: string
           description: Specifies which LDAP attribute is used for the subject identifier claim.
           example: inum
+        publicSubjectIdentifierPerClientEnabled:
+          type: boolean
+          description: Specifies whether public subject identifier is allowed per client.
+        subjectIdentifiersPerClientSupported:
+          type: array
+          description: A list of the subject identifiers supported per client.
+          items:
+            type: string
+          enum:
+            - mail
+            - uid			
         responseTypesSupported:
```

Comment:
so its an array not an array of strings like what is showing in the server?

Reply:
`subjectIdentifiersPerClientSupported` as per code is list of String and hence represented as array of String in spec.
``` private **List<String>** subjectIdentifiersPerClientSupported;```

![image](https://user-images.githubusercontent.com/43700552/186415823-97bdb34b-299b-4376-96f7-c5d7597116e6.png)